lerobot/scripts/push_dataset_to_hub.py

```diff
@@ -245,10 +243,16 @@ def main():
     parser = argparse.ArgumentParser()
 
     parser.add_argument(
-        "--data-dir",
+        "--raw-dir",
+        type=Path,
+        required=True,
+        help="Directory containing input raw datasets (e.g. `data/aloha_mobile_chair_raw` or `data/pusht_raw).",
+    )
+    parser.add_argument(
+        "--out-dir",
         type=Path,
         required=True,
```

Comment:
TODO for me:

For output directory, we might want to set `required=False` and `default=None`.
We shouldn't need to specify it if we only pusht to hub.

Currently we might use output dir for 2 things:
- a temporary directory to save images and convert them to videos
- the final directory containing a local dataset

Instead, of using output dir as a temporary directory, we should probably use a directory inside raw dir.

Reply:
DOne